quartermaster/src/main/java/com/github/reflectoring/infiniboard/quartermaster/dashboard/rest/DashboardController.java

```diff
@@ -0,0 +1,59 @@
+package com.github.reflectoring.infiniboard.quartermaster.dashboard.rest;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.web.PageableDefault;
+import org.springframework.data.web.PagedResourcesAssembler;
+import org.springframework.hateoas.PagedResources;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.github.reflectoring.infiniboard.quartermaster.dashboard.domain.Dashboard;
+import com.github.reflectoring.infiniboard.quartermaster.dashboard.domain.DashboardRepository;
+
+import static org.springframework.http.HttpStatus.NOT_FOUND;
+import static org.springframework.http.HttpStatus.OK;
+import static org.springframework.web.bind.annotation.RequestMethod.GET;
+
+@RestController
+@RequestMapping("/api/dashboards")
+public class DashboardController {
+
+    private DashboardRepository repository;
+
+    private DashboardResourceAssembler dashboardResourceAssembler;
+
+    @Autowired
+    public DashboardController(DashboardRepository repository,
+                               DashboardResourceAssembler dashboardResourceAssembler) {
+        this.repository = repository;
+        this.dashboardResourceAssembler = dashboardResourceAssembler;
+    }
+
+    @SuppressWarnings("unchecked")
+    @RequestMapping(method = GET)
+    public ResponseEntity<PagedResources<DashboardResource>> getAllDashboards(@PageableDefault Pageable pageable,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUndocumentedApi)